espnet/bin/mt_train.py

```diff
@@ -222,6 +240,9 @@ def main(cmd_args):
             else:
                 ngpu = len(p.stderr.decode().split('\n')) - 1
     else:
+        if is_torch_1_2_plus:
+            assert args.ngpu == 1, "There are some bugs with multi-GPU processing in PyTorch 1.2+" \
```

Comment:
because this bug only related to cudnn RNN (flatten params?), this is annoying for other nnets.  I think it can be logging.info or something just useful for users